src/main/resources/application.conf

```diff
@@ -133,14 +137,64 @@ akka.persistence.cassandra {
   snapshot.tables-autocreate = on
 
   cleanup {
-   dry-run = false
+    dry-run = false
   }
 }
 
 datastax-java-driver {
   advanced.reconnect-on-init = on
 }
 
+
+akka-persistence-jdbc {
+  shared-databases {
+    slick {
+      profile = "slick.jdbc.PostgresProfile$"
+      db {
+        host = "localhost"
+        host = ${?DB_HOST}
+        url = "jdbc:postgresql://"${akka-persistence-jdbc.shared-databases.slick.db.host}":5432/docker?reWriteBatchedInserts=true"
+        user = "docker"
+        password = "docker"
+        driver = "org.postgresql.Driver"
+        numThreads = 5
+        maxConnections = 5
+        minConnections = 1
+      }
+    }
+  }
+}
+
+jdbc-journal {
+  use-shared-db = "slick"
+}
+
+# the akka-persistence-snapshot-store in use
+jdbc-snapshot-store {
+  use-shared-db = "slick"
+}
+
+# the akka-persistence-query provider in use
+jdbc-read-journal {
+  use-shared-db = "slick"
+}
```

Comment:
would be nice to move to the same config structure as in the guide examples: https://akka.io/platform-guide/wip/microservices-tutorial/dev-env.html#_downloadable_example_source

Reply:
Done! And integrated with the operator